### PR TITLE
Pattern Loading indicator

### LINKED ISSUE #65: Feature/Loading Indicator
Loading indicator component when we don't know percent done (should use #63 when we do)

### PR DESCRIPTION
**Pull request by ryanbrownhill**
Resolves #65 

**CHANGELOG**
- :memo: Add Readme.md for the usage of pattern
- Add IMG files for loader
- Add Modernizr JS with SMIL & SVG detection
- Add HTML with base loading indicator pattern
- Add Sass for styling and fallbacks of loader

### CODE REVIEW

**Comment by ryanbrownhill** [comment]
@Snugug @britanyponvelle and I don't know why Travis is Erroring....

**Comment by Snugug** [comment]
Because it's having problems installing our Node dependencies. I restarted it

File: patterns/components/loading-indicator/loading-indicator.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div class="loader">
+  <object type="image/svg+xml" data="img/loading-indicator.svg" class="loader--svg"></object>
```
Comment (Snugug) [inline patterns/components/loading-indicator/loading-indicator.html]: instead of `img`, let's use `images`

**Comment by Snugug** [comment]
Have you tested this in the actual pattern library? The URL will be (when you load up the runner) is:

localhost:3000/ui-patterns/patterns/components/loading-indicator

**Comment by ryanbrownhill** [comment]
OMG TRAVIS! :clap: :smile:

**Comment by ryanbrownhill** [comment]
This is ready for :+1: 's!

**Comment by bobbinrobyn** [comment]
Yay! :+1:

**Comment by britanyponvelle** [comment]
:+1:

**Comment by ryanbrownhill** [comment]
@IBM-Watson/design-acceleration :+1: ?

**Comment by poofichu** [comment]
Bueno! :+1:

**Comment by poofichu** [comment]
Resolves #65

**Pull request merged**